### ISSUE TITLE
legal risk, experience, and other advice

### ISSUE THREAD

**Issue by EmeraldSnorlax** [Nikilites/nuzu#11]
hi, i just want to echo the sentiment of the other issues here, but perhaps more gently.

yuzu was taken down by, nintendo, a notoriously litigious company. not only that, but you don't even own the rights to the original contributions. if nothing else, you are exposing yourself to potentially life ruining legal issues due to acting as an individual rather than under the shelter of an llc.

also, more pragmatically: with all due respect, this fork isn't going to go anywhere. right now, you don't have much experience (which is fine! we all started doing silly things!), but i dont think you realise how monumental of a task writing and maintaining an emulator for a modern console is.

plus, simply creating a repo and asking for developers to help you, then claiming that youre the new owner, is, gently, very frowned upon -- especially when looking at this repo and your profile, you don't really have any development experience, or even know how to use git.

i appreciate youre young and don't fully understand everything that's going on. perhaps in the future, when you learn more, you truly can pick up a torch and write a switch emulator. but right now is not that time for the reasons mentioned above.

**Comment by nekoworkshop** [comment]
The owner has removed the "Almost 15" from their bio lol 
![Screenshot_2024-03-05-10-07-00-223_com brave browser](https://github.com/Nikilites/nuzu/assets/1346257/abe4cdad-8911-4e56-a60a-057df61038f9)

**Comment by Nikilites** [comment]
need to do it for the hate

**Comment by EmeraldSnorlax** [comment]
i dont think mocking their age is helpful, as we all have done incredibly stupid things at that age. i hope you dont let the hate get to you, but see the truth in what they are saying

**Comment by philpax** [comment]
Agreed with this; I was going to express this statement more forcefully, but I'm glad you've taken a more gentle approach.

Writing and maintaining an emulator is an extremely complex project that requires a great deal of technical expertise and opens you up to significant legal risk.

Please shut this repository down before you cause problems for the wider community *and* yourself.

**Comment by SeleDreams** [comment]
> hi, i just want to echo the sentiment of the other issues here, but perhaps more gently.
> 
> yuzu was taken down by, nintendo, a notoriously litigious company. not only that, but you don't even own the rights to the original contributions. if nothing else, you are exposing yourself to potentially life ruining legal issues due to acting as an individual rather than under the shelter of an llc against a company that has a history of literally [placing people that crossed them under effectively indentured servitude](https://www.ign.com/articles/nintendo-hacker-gary-bowser-released-from-prison-still-owes-millions).
> 
> also, more pragmatically: with all due respect, this fork isn't going to go anywhere. right now, you don't have much experience (which is fine! we all started doing silly things!), but i dont think you realise how monumental of a task writing and maintaining an emulator for a modern console is.
> 
> plus, simply creating a repo and asking for developers to help you, then claiming that youre the new owner, is, gently, very frowned upon -- especially when looking at this repo and your profile, you don't really have any development experience, or even know how to use git.
> 
> i appreciate youre young and don't fully understand everything that's going on. perhaps in the future, when you learn more, you truly can pick up a torch and write a switch emulator. but right now is not that time for the reasons mentioned above.

When it comes to "owning the rights to the contributions" Yuzu was GPL licensed so this is a non-issue. There are no legal issues to taking the source of a version that was gpl licensed. It happened a lot over the years for lots of softwares that left the open source.

Now though, I do recommend for devs who want to take over, to do the development in European countries such as France where the interoperability laws definitely allow to reverse engineer to guarantee interoperability of software. As such, using software from a machine on another. Also the fact there are no software patents in France.

**Comment by EmeraldSnorlax** [comment]
> When it comes to "owning the rights to the contributions" Yuzu was GPL licensed so this is a non-issue.

i'm not sure this is not true. the bulk of contributions came from tropic haze, who owned the copyright. part of the settlement involves [not distributing yuzu in any form](https://storage.courtlistener.com/recap/gov.uscourts.rid.56980/gov.uscourts.rid.56980.10.0.pdf), and the developers themselves have stated this intent. 

would this not taint all downstream yuzu forks?

**Comment by EmeraldSnorlax** [comment]
yes, i agree that in practice they can't practically take back the existing gpl code. but nintendo would have a case to go after this repo for copyright infringement too if they were successful on yuzu, considering this is *literally* just a reupload

**Comment by SeleDreams** [comment]
> > When it comes to "owning the rights to the contributions" Yuzu was GPL licensed so this is a non-issue.
> 
> i'm not sure this is true. the bulk of contributions came from tropic haze, who owned the copyright. part of the settlement involves [not distributing yuzu in any form](https://storage.courtlistener.com/recap/gov.uscourts.rid.56980/gov.uscourts.rid.56980.10.0.pdf), and the developers themselves have stated this intent.
> 
> would this not taint all downstream yuzu forks?
> 
> now, it obviously isn't this simple as "license-takesies-backsies" in practice, but do you _really_ want to play chicken with nintendo's lawyers, when this repo is literally just a verbatim copy of the code that got yuzu into this mess?

For the second part. Here I wasn't specifically defending this exact fork as to begin with it was most likely a troll or a kid, but either way if someone was to take over like I sais they would definitely need to cover all bases legally by developping in countries that protect their rights and by making sure to analyze all risky areas and patching them out before sharing the new fork

**Comment by SeleDreams** [comment]
> yes, i agree that in practice they can't practically take back the existing gpl code. but nintendo would have a case to go after this repo for copyright infringement too if they were successful on yuzu, considering this is _literally_ just a reupload

Yuzu was a settlement with the developers outside of court. Not a court decision, so they won't have jurisprudence
